library/src/morpheus-master/sass/modules/tool/conversations/_sakai-post.scss

```diff
@@ -21,14 +21,14 @@
 
   .post-created-date {
     font-size: 14px;
-    color: #666666;
+    color: var(--sakai-text-color);
```

Comment:
You may consider `--sakai-text-color-dimmed` but I think what you've got works as well.